tests/src/JIT/HardwareIntrinsics/X86/Shared/GenerateTests.csx

```diff
@@ -20,262 +20,295 @@ using System.IO;
 
 private static readonly (string templateFileName, string[] templateData)[] SseInputs = new []
 {
-    // TemplateName                              Isa,   LoadIsa, Method,                                     BaseType, VectorType,  VectorSize, NextValue,                      ValidateFirstResult,                                                                               ValidateRemainingResults
```

Comment:
Other changes in here are to support the additional metadata entries in the templates.